### PR TITLE
correct document of using br backup and restore system tables

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6057
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
-### 系统库下表的备份与恢复
+### ###在 `mysql` 系统表中备份和恢复表数据（实验特性）
```
Comment (IANTHEREAL) [inline br/backup-and-restore-tool.md:92]:
```suggestion
### ###备份和恢复 `mysql` 系统库下面的数据（实验特性）
```

**Comment by IANTHEREAL** [comment]
Rest LGTM

**Review by 3pointer — APPROVED** [review]

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
 | 用 BR v4.0 备份 TiDB v4.0 | ✅ | ✅  | ✅（如果 TiKV >= v4.0.0-rc.1，BR 包含 [#233](https://github.com/pingcap/br/pull/233) Bug 修复，且 TiKV 不包含 [#7241](https://github.com/tikv/tikv/pull/7241) Bug 修复，那么 BR 会导致 TiKV 节点重启) |
 | 用 BR nightly 或 v5.0 备份 TiDB v4.0 | ❌（当 TiDB 版本小于 v4.0.9 时会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) | ❌（当 TiDB 版本小于 v4.0.9 会出现 [#609](https://github.com/pingcap/br/issues/609) 问题) |
 
-### 系统库下表的备份与恢复
+### ###备份和恢复 `mysql` 系统库下面的数据（实验特性）
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:92]:
```suggestion
### 备份和恢复 `mysql` 系统库下的表数据（实验特性）
```

File: br/backup-and-restore-tool.md [hunk 3]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
 
-在 v5.1.0 之前，BR 备份时会过滤掉系统库 (`mysql.*`) 的数据。
+>**警告：**
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:94]:
```suggestion
> **警告：**
```

File: br/backup-and-restore-tool.md [hunk 4]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
+>
+> 该功能目前为实验性功能，未经彻底测试。**不建议**在生产环境中使用该功能。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:96]:
```suggestion
> 该功能目前为实验性功能，尚未经过充分测试。**不建议**在生产环境中使用该功能。
```

File: br/backup-and-restore-tool.md [hunk 5]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
 
-自 v5.1.0 起，BR 默认备份集群内的全部数据，包括系统库 (`mysql.*`)，但为了兼容 v5.1.0 之前的 BR 版本，恢复数据的时候默认**不恢复**系统库下的数据。只有设置了 [`filter` 参数](/br/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据)才会把系统库下的数据恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
+在 v5.1.0 之前，BR 备份时会过滤掉系统库 (`mysql.*`) 的数据。自 v5.1.0 起，BR 默认**备份**集群内的全部数据，包括系统库 (`mysql.*`)。但是由于在 `mysql.*` 中恢复系统表的技术实现尚未完成，因此系统库 `mysql` 中的表默认**未恢复**。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:98]:
```suggestion
在 v5.1.0 之前，BR 备份时会过滤掉系统库 `mysql.*` 的表数据。自 v5.1.0 起，BR 默认**备份**集群内的全部数据，包括系统库 `mysql.*` 中的数据。但由于恢复 `mysql.*` 中系统表数据的技术实现尚不完善，因此 BR 默认**不恢复**系统库 `mysql` 中的表数据。
```

File: br/backup-and-restore-tool.md [hunk 6]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
 
-BR 之所以对系统表进行如上处理，是因为用户可以在系统库中创建非系统表，并且有需求对这些非系统表进行备份恢复。同时，BR 对系统表的恢复功能尚不完善，所以你在实际操作中，**不建议**恢复集群本身存在的系统表。
+如果希望将系统表（如 `mysql.usertable1`）的数据还原到系统库 `mysql`，可以设置 [`filter` 参数](/br/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据) 来过滤表名（`-f“mysql.usertable1`）。设置后，系统库下的数据会恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:100]:
```suggestion
如果你希望将系统表（如 `mysql.usertable1`）的数据恢复到系统库 `mysql` 中，可以设置 [`filter` 参数](/br/use-br-command-line-tool.md#使用表库过滤功能备份多张表的数据) 来过滤表名 (`-f "mysql.usertable1"`)。完成设置后，系统库下的表数据会恢复到临时库中，然后通过对临时库表进行重命名的方式恢复到系统库。
```

File: br/backup-and-restore-tool.md [hunk 7]
```diff
@@ -89,15 +89,17 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
 
-因此，在恢复数据涉及到系统库的时候，你需要通过**表名过滤** (`-f "mysql.usertable1"`) 的方式过滤具体的表，以此来进行数据恢复。同时 BR 默认提供一个黑名单，用来阻止恢复以下系统表。即使你指定了 `-f "mysql.*"`，以下表也不会被恢复：
+由于技术原因，无法正确恢复以下系统表。即使你指定了 `-f "mysql.*"`，以下表也不会被恢复：
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:102]:
```suggestion
由于技术原因，以下系统表不能被正确恢复。即使你指定了 `-f "mysql.*"`，以下表也不会被恢复：
```

**Pull request merged**